### PR TITLE
Add Gamma and Geary C processes for spatial autocorrelation

### PR DESCRIPTION
**Pull request by chuehlien**

### CODE REVIEW

**Comment by mbertrand** [comment]
Thanks @chuehlien , can you run:

```
flake8 --config tests/flake8.cfg gaia_spatialstats tests
```

and fix the lines that it complains about?

I see some new test data but not new test cases yet, maybe test_parser.py and/or test_processes.py need a commit & push?

**Comment by chuehlien** [comment]
Thanks @mbertrand! I've added the tests and fixed the flake8 complaints.

**Comment by mbertrand** [comment]
Thanks @chuehlien, looks good, I'm going to assign the PR to @aashish24 for review.

File: gaia_spatialstats/processes.py [hunk 1]
```diff
@@ -198,8 +198,143 @@ def compute(self):
         self.output.write()
         logger.debug(self.output)
 
+
+class GearyCProcess(GaiaProcess):
+    u"""
```
Comment (aashish24) [inline gaia_spatialstats/processes.py:203]: @chuehlien @mbertrand is this u necessary?

File: tests/cases/test_parser.py [hunk 2]
```diff
@@ -77,6 +77,50 @@ def test_process_autocorrelation(self):
             if process:
                 process.purge()
 
+    def test_process_gamma(self):
```
Comment (aashish24) [inline tests/cases/test_parser.py]: @chuehlien @mbertrand i am wondering why we are adding these tests to test_parser.py file. Shouldn't we add them to the test_processes.py?

File: tests/data/gamma_process.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "_type": "gaia_spatialstats.processes.GammaProcess",
+  "inputs": [
+    {
+      "name": "input",
+      "_type": "gaia.geo.geo_inputs.VectorFileIO",
+      "uri": "{basepath}//baghdad_hospitals.geojson"
```
Comment (aashish24) [inline tests/data/gamma_process.json:7]: is baghdad_hospitals.geojson checked in as well?

**Comment by aashish24** [comment]
Mostly looks good to me, just has some minor comments. Please have a look, it is possible that I missed something. thanks.

File: gaia_spatialstats/processes.py (hunk 1 above)
Comment (chuehlien) [reply gaia_spatialstats/processes.py:203]: I've removed the u. One of the apostrophe in there was a unicode character but that has been removed.

File: tests/data/gamma_process.json (hunk 3 above)
Comment (chuehlien) [reply tests/data/gamma_process.json:7]: It's checked in.

File: gaia_spatialstats/processes.py (hunk 1 above)
Comment (aashish24) [reply gaia_spatialstats/processes.py:203]: okay thanks @chuehlien

**Comment by aashish24** [comment]
LGMT 👍

**Pull request merged**